usb/esp_tinyusb/tusb_msc_storage.c

```diff
@@ -350,6 +380,13 @@ esp_err_t tinyusb_msc_storage_init_sdmmc(const tinyusb_msc_sdmmc_config_t *confi
     s_storage_handle->base_path = NULL;
     s_storage_handle->card = config->card;
 
+    /* Callbacks setting up*/
+    if (config->callback_mount_changed) {
+        tinyusb_msc_register_callback(TINYUSB_MSC_EVENT_MOUNT_CHANED, cfg->callback_mount_changed);
```

Comment:
`tinyusb_msc_register_callback(TINYUSB_MSC_EVENT_MOUNT_CHANED, cfg->callback_mount_changed);`
->
`tinyusb_msc_register_callback(TINYUSB_MSC_EVENT_MOUNT_CHANED, config->callback_mount_changed);`

Reply:
@larryli Thanks for the PR. Please check my comments.

Reply:
Sorry, I Got it.

Reply:
> `tinyusb_msc_register_callback(TINYUSB_MSC_EVENT_MOUNT_CHANED, cfg->callback_mount_changed);` -> `tinyusb_msc_register_callback(TINYUSB_MSC_EVENT_MOUNT_CHANED, config->callback_mount_changed);`

Hey @larryli ,
Have you fixed this one ? I am not able to find it !!
Please check once more

Reply:
Hey @larryli 

Sorry, I am not able to find out the changes for this comment.
Please check if you have fixed it.